lib/ibm_watson/speech_to_text_v1.rb

```diff
@@ -920,11 +948,22 @@ def check_jobs
     #   **Note:** Applies to US English, Japanese, and Korean transcription only.
     #
     #   See [Numeric
-    #   redaction](https://cloud.ibm.com/docs/services/speech-to-text/output.html#redaction).
+    #   redaction](https://cloud.ibm.com/docs/services/speech-to-text?topic=speech-to-text-output#redaction).
+    #
+    #   The parameter accepts a minimum value of 0.1 seconds. The level of precision is
+    #   not restricted, so you can specify values such as 0.25 and 0.125.
+    #
+    #   The service does not impose a maximum value. If you want to receive processing
+    #   metrics only for transcription events instead of at periodic intervals, set the
+    #   value to a large number. If the value is larger than the duration of the audio,
+    #   the service returns processing metrics only for transcription events.
+    # @param audio_metrics [Boolean] If `true`, requests detailed information about the signal characteristics of the
+    #   input audio. The service returns audio metrics with the final transcription
+    #   results. By default, the service returns no audio metrics.
     # @param content_type [String] The format (MIME type) of the audio. For more information about specifying an
     #   audio format, see **Audio formats (content types)** in the method description.
     # @return [IBMCloudSdkCore::DetailedResponse] A `IBMCloudSdkCore::DetailedResponse` object representing the response.
-    def create_job(audio:, model: nil, callback_url: nil, events: nil, user_token: nil, results_ttl: nil, language_customization_id: nil, acoustic_customization_id: nil, base_model_version: nil, customization_weight: nil, inactivity_timeout: nil, keywords: nil, keywords_threshold: nil, max_alternatives: nil, word_alternatives_threshold: nil, word_confidence: nil, timestamps: nil, profanity_filter: nil, smart_formatting: nil, speaker_labels: nil, customization_id: nil, grammar_name: nil, redaction: nil, content_type: nil)
+    def create_job(audio:, model: nil, callback_url: nil, events: nil, user_token: nil, results_ttl: nil, language_customization_id: nil, acoustic_customization_id: nil, base_model_version: nil, customization_weight: nil, inactivity_timeout: nil, keywords: nil, keywords_threshold: nil, max_alternatives: nil, word_alternatives_threshold: nil, word_confidence: nil, timestamps: nil, profanity_filter: nil, smart_formatting: nil, speaker_labels: nil, customization_id: nil, grammar_name: nil, redaction: nil, audio_metrics: nil, content_type: nil)
```

Comment:
We need `processing_metrics` and `processing_metrics_interval` in `create_job`. We only remove it from async recognize. Same question about positional params here for `content-type` as well.